easybuild/easyblocks/generic/systemcompiler.py

```diff
@@ -216,14 +209,19 @@ def make_module_req_guess(self):
         generate_standalone_module parameter is False
         """
         if self.cfg['generate_standalone_module']:
-            if self.cfg['name'] in ['GCC','GCCcore']:
-                guesses = EB_GCC.make_module_req_guess(self)
-            elif self.cfg['name'] in ['icc']:
-                guesses = EB_icc.make_module_req_guess(self)
-            elif self.cfg['name'] in ['ifort']:
-                guesses = EB_ifort.make_module_req_guess(self)
+            if self.compiler_prefix in ['/usr', '/usr/local']:
+                # Force off adding paths to module since unloading such a module would be a potential shell killer
+                self.log.warning("Ignoring option 'generate_standalone_module' since installation prefix is %s",
+                                 self.compiler_prefix)
             else:
-                raise EasyBuildError("I don't know how to generate module var guesses for %s", self.cfg['name'])
+                if self.cfg['name'] in ['GCC','GCCcore']:
```

Comment:
This should not be indented under `else`, should always be done regardless of `compiler_prefix` in `/usr`?

Reply:
nvm, that's the whole point of the `/usr` check :)